86BoxManager/Properties/AssemblyInfo.cs

```diff
@@ -31,5 +31,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 
-[assembly: AssemblyVersion("1.7.25.20330")]
-[assembly: AssemblyFileVersion("1.7.25.20330")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.7.27.20330")]
```

Comment:
Please fix the assembly version so it complies with the guidelines (see project's wiki page).